docs/MAINTAIN.md

```diff
@@ -0,0 +1,16 @@
+# Supporting new families
+
+## Write device file
+
+a file <family>.cmake must be added to the folder cmake/stm32
+This file containes the differents devices and the regex used to parse them.
+It also give information on the RAM and CCRAM available
+
+## Add family to list of know families
+
+Update the list `STM32_SUPPORTED_FAMILIES_LONG_NAME` located in the cmake/stm32/common.cmake file
+
+
```

Comment:
## Update cube, cmsis and hal version in utilities.cmake

The versions to use can be found as follow:
 - Cube version : a valid tag from the repo https://github.com/STMicroelectronics/STM32Cube${FAMILY}
 - Cmsis version : a valid tag from the repo https://github.com/STMicroelectronics/cmsis_device_${FAMILY_L}
 - Hal version : a valid tag from the repo https://github.com/STMicroelectronics/stm32${FAMILY_L}xx_hal_driver